docs/tools/eip191-signerjs/getting-started.md

```diff
@@ -2,7 +2,7 @@
 sidebar_position: 1
 ---
 
-# Getting Started
+# Getting Started with EIP191-signer.js
```

Comment:
Suggest reverting the title change. It seems unecessary as we are in the Getting Started page of the specific tool already. The user already clicked on the Tool to get to this page, so it knows the tool. No need to re-mention it.